server/vulnerabilities/cve.go

```diff
@@ -61,7 +61,7 @@ var (
 
 	// max age to be considered a recent vulnerability (relative to NVD's published date)
 	// (a var to be able to change in tests)
-	recentVulnMaxAge = 2 * 24 * time.Hour
+	recentVulnMaxAge = 30 * 24 * time.Hour
```

Comment:
Is this necessary due to the "latency" from "CVE published" until "CVE present in NVD"?
(For future reference, maybe comment why we settled to 30 days.)

Reply:
This is the original ticket https://github.com/fleetdm/fleet/issues/4879, not sure if it's because of that latency, or based on how this works day to day, @GuillaumeRoss ?

Reply:
My original post which was edited by a benevolent product @noahtalerman person asked for it to be customizable, but I think going for a fixed value is the easiest way to improve this fast.

The problem is because there is latency between published and present indeed, and we settled for a month because that usually makes sense for vulnerability management. Companies like Microsoft, Adobe, etc, have fixed patch release schedules (unless emergency) that are once a month, so I figure even if a vuln is 2 weeks old it's still within the latest "cycle".